tests/format/scss/print-width/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,79 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`print-width.scss - {"printWidth":40} format 1`] = `
+====================================options=====================================
+parsers: ["scss"]
+printWidth: 40
+                                        | printWidth
+=====================================input======================================
+@use "library" with ($black: #222, $border-radius: 0.1rem, $font-family: "Helvetica, sans-serif", $list: (1, 2, 3));
+
+@forward "library" with ($black: #222, $border-radius: 0.1rem, $font-family: "Helvetica, sans-serif", $list: (1, 2, 3));
+
+=====================================output=====================================
+@use "library" with
+  (
+    $black: #222,
+    $border-radius: 0.1rem,
+    $font-family:
+      "Helvetica, sans-serif",
```

Comment:
hm , I think it should on the same line...

Reply:
It’s whole string so it’ placed in new line when line width is 40 characters or less. [Try playing with print width](https://deploy-preview-11516--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEABArgZzgAgDogA2AlgEYBOAhuQJ4E4DuxMAFjgBQAkphlYA1khwBiAEziANDm4RyAEzjkAtFTnEsQgAwA6AIzk4AWymcAZtBhLTlQ8UI0hBABJxCANzgxiYSlMyUoTCVscmJTAhNIdFhFTCF2XSlRKQBmAEo0gG48KBAJEAgABy9oTGRQanIIBgAFagQylEpCBkoaMvyKPn5PAGUbOAAZYig4ZGtCbE6qAT7CvhGAc2QYcnQ4fKNSODkFOUGAxfRKRbgAMVlDShgvKGWm9BgIPJAWGENCAHUWZjhMebAcF6DWYxDczBoyHAmA6IBGIRgNSoiyu42aUxAACtMAAPXpLQhwACK6Ag8DRkw2IHm5BCUMwYBhL0KoVgn2IclYyBSAHZ8iyINhPlRClCWX9FB4XgBHUnwJFFRogShBUY7HYvAyy4gGJEnVFICYY7C2FZrKmYAnEuVjQ3oqkwSikdmcljIZIgVaUOxLADCEEMBpAfwArC8sHAACpOxpGqludYASSgClgvTAoWKAEEU70YDRCRTsABfYtAA).